megalinter/reporters/UpdatedSourcesReporter.py

```diff
@@ -69,14 +69,18 @@ def produce_report(self):
                 if apply_fixes.lower() != "none":
                     try:
                         repo = git.Repo(os.path.realpath(self.master.github_workspace))
+                        repo.config_writer().set_value("user", "name", "MegaLinter").release()
+                        repo.config_writer().set_value("user", "email", "megalinter@megalinter.io").release()
                         repo.git.add(update=True)
                         repo.git.commit('-m', 'megalinter auto fixes')
                         repo.git.push()
                     except git.GitCommandError as giterr:
                         logging.error(
                             "[Updated Sources Reporter] Failed to git push auto fixes: " + str(giterr.stderr) + "\n"
-                            "Download it from artifacts then copy-paste it in your local repo to apply linters updates"
+                            "[Updated Sources Reporter] Download fixed source files from artifacts "
+                            "then copy-paste into your repo to apply linters updates"
                         )
+                        self.master.status = "error"
```

Comment:
@nvuillam is this line correct in order to set megalinter job to fail (if the git commands fail)?

Reply:
@lukelloyd1985 plz don't make MegaLinter fail if a reporter does not work ^^
I prefer a warning message :)

Reply:
Hmmm ok but issue is unless MegaLinter fails the comment on the PR will be resolved rather than active. Unless we can change that so PR comment is active if MegaLinter has warnings?

Reply:
How about that?

Reply:
There are many cases where git will fail (we saw that with other git operations, like git pull , when git credentials are not kept, or git rights are missing), so... I really prefer to not fail MegaLinter in that case , because the impact on existing pipelines would be catastrophic ^^

Sorry but i think that the less impacting update is if the person setting up MegaLinter carefully checks the logs (reporter ones are at the end) if APPLY_FIXES is activated :/

Reply:
Yeah that makes sense. I've already changed it on the last commit so it sets the status to warning instead. I've also changed the Azure PR comment so it sets the comment as active if MegaLinter doesn't complete successfully